### ISSUE TITLE
Reformat firmware source to have leading tabs and good indents

### ISSUE THREAD

**Issue by jschrempp** [SISProject/SISSoftware#51]
We need to pretty up the code, but after we release since we have tested this version of firmware a lot.

**Comment by SISProject** [comment]
Agreed - after the release, when we have time to perform some solid regression testing.  Since these are just cosmetic changes, it can be version 1.01.

Thanks,
Bob

-----Original Message-----
